programs/mpl-staking/src/instructions/penalties/restrict_batch_minting.rs

```diff
@@ -17,17 +13,13 @@ pub fn restrict_batch_minting(
         MplStakingError::InvalidRealmAuthority
     );
 
-    let signers_seeds = registrar_seeds!(&registrar);
+    require!(
+        until_ts > clock_unix_timestamp(),
```

Comment:
I think we could drop this check as a way to remove the restriction. The use case is to restrict a user for a month and then remove the restriction once the user is cleared

Reply:
It's a typo. It rather should've been `until_ts < clock_unix_timestamp()` in order not to set the time that is lower than the current one.